### PR TITLE
Endpoint for get places

### PR DESCRIPTION
**Pull request by Erol0019**
har oprettet endpoint for get places

### CODE REVIEW

File: rpg_api/tests/pytest/endpoint/place/test_get_places.py [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+    user = await factories.BaseUserFactory.create()
+    header = test_utils.get_user_header(user.id)
+
+    response = await client.get(url, headers=header)
+
+    assert response.status_code == status.HTTP_200_OK
+    response_data = test_utils.get_data(response)
+
+    assert isinstance(response_data, list)
+    assert len(response_data) == 0
```
Comment (martinlaursen97) [inline rpg_api/tests/pytest/endpoint/place/test_get_places.py:81]: And then this test would not be needed either

File: rpg_api/tests/pytest/endpoint/place/test_get_places.py [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+    assert response.status_code == status.HTTP_200_OK
+
+    response_data = test_utils.get_data(response)
+    assert isinstance(response_data, list)
+    assert len(response_data) == 1
+
+    assert response_data[0]["name"] == name_place
+    assert response_data[0]["radius"] == radius
+    assert response_data[0]["x"] == x_place
+    assert response_data[0]["y"] == y_place
```
Comment (martinlaursen97) [inline rpg_api/tests/pytest/endpoint/place/test_get_places.py:41]: You don't need to test if the PlaceFactory is creating the place object correctly, since that is not in the scope of this test. It should only test the behaviour of the `GET /places` endpoint.

Since the endpoint does not require authentication, we don't need to send authentication headers when making the request. For this reason, we also don't need to create a user for the test.

The test could be rewritten like this, which also tests for different states of the database (empty database, has 1 place, has multiple places)
```suggestion
@pytest.mark.anyio
@pytest.mark.parametrize(
    "create_num",
    [
        0,  # empty
        1,  # single
        2,  # multiple
    ],
)
async def test_get_all_places(client: AsyncClient, create_num: int) -> None:
    """Test get all places: 200."""

    # x = i is used to avoid overlapping places
    places = [await factories.PlaceFactory.create(x=i) for i in range(create_num)]

    response = await client.get(url)
    assert response.status_code == status.HTTP_200_OK

    response_data = get_data(response)
    assert len(response_data) == create_num

    for index, place in enumerate(places):
        assert response_data[index]["id"] == str(place.id)
        assert response_data[index]["name"] == place.name
        assert response_data[index]["description"] == place.description
        assert response_data[index]["x"] == place.x
        assert response_data[index]["y"] == place.y
        assert response_data[index]["radius"] == place.radius
```
